src/function.py

```diff
@@ -109,7 +109,7 @@ def zip_and_upload_folder(client: CogniteClient, folder: str, name: str) -> int:
     raise FunctionDeployError(f"Failed to upload file ({name}) to CDF Files")
 
 
-@retry(exceptions=(FunctionDeployTimeout, FunctionDeployError), tries=5, delay=2, jitter=2)
+@retry(exceptions=(IOError, FunctionDeployTimeout, FunctionDeployError), tries=5, delay=2, jitter=2)
```

Comment:
Functions already try 5 times over a period of 31 seconds, is FilesAPI really this "unstable"? 😞

Reply:
My goal to make our part to be as friendly as possible. so I look at that from the point of not having many people complaining to me.

Reply:
I don't disagree ;)